### DISCUSSION TITLE
ReactFlow error

### DISCUSSION THREAD

**Discussion by rasti8** [originjs/vite-plugin-federation#418] [Q&A]
Hi, 
I'm facing an issue related with reactflow and can´t understand why. Basically in my remote app I'm using this library and when I try to call it in the host I always get this error.

![2023-05-12-16-24-06](https://github.com/originjs/vite-plugin-federation/assets/83433150/745737bc-b269-4a12-8682-376f09f33a36)

Can anyone help me understand the reason of this? 
Thanks!

**Comment by VladimirTambovtsev** [comment]
I have the same error in the latest version. I'm not sure, but probably because Reactflow component is wrapped in forwardRef.
I can confirm that with webpack module federation (host and remote) reactflow works fine.

Would be great if it can be fixed. This bug blocks us to use vite-plugin-federation but we don't want to switch back to webpack.
